### PR TITLE
[NON FUNCTIONAL CHANGE] Fix spelling, punctuation, and some end of line in onnx-mlir help

### PR DESCRIPTION
**Pull request by AlexandreEichenberger**
fix spelling, punctuation in `onnx-mlir -h` generated text.

not perfect but better.

### CODE REVIEW

**Review by Sunny-Anand — APPROVED** [review]
LGTM

**Comment by AlexandreEichenberger** [comment]
Since there is no functional changes, I will wait for having a review, respond to comments if any, and then update to the latest version before merging the PR.

**Pull request merged**
